fpdf/fpdf.py

```diff
@@ -989,6 +989,23 @@ def ellipse(self, x, y, w, h, style=None):
             )
         )
 
+    @check_page
+    def circle(self, x, y, r, style=None):
+        """
+        Outputs a circle.
+        It can be drawn (border only), filled (with no border) or both.
+
+        Args:
+            x (int): Abscissa of upper-left bounging box.
+            y (int): Ordinate of upper-left bounging box.
+            r (int): Radius of the circle.
+            style (int): Style of rendering. Possible values are:
+                * `D` or empty string: draw border. This is the default value.
```

Comment:
Could you replace "empty string" by `None`? This would be slightly more accurate ;)